src/app/components/PriceChart.tsx

```diff
@@ -4,30 +4,52 @@ import {
   CANDLE_PERIODS,
   OHLCVData,
   setCandlePeriod,
+  handleCrosshairMove,
+  // fetchCandlesForInitialPeriod,
+  initializeLegend,
 } from "../redux/priceChartSlice";
 import { useAppDispatch, useAppSelector } from "../hooks";
+import { formatPercentageChange } from "../utils";
+import { displayAmount } from "../utils";
 
 interface PriceChartProps {
   data: OHLCVData[];
 }
 
 function PriceChartCanvas(props: PriceChartProps) {
   const chartContainerRef = useRef<HTMLDivElement>(null);
+  const dispatch = useAppDispatch();
   const { data } = props;
+
   useEffect(() => {
     const chartContainer = chartContainerRef.current;
 
+    // dispatch(fetchCandlesForInitialPeriod());
+    if (data && data.length > 0) {
+      dispatch(initializeLegend());
+    }
+
     if (chartContainer) {
       const handleResize = () => {
         chart.applyOptions({ width: chartContainer.clientWidth });
       };
 
       const chart = createChart(chartContainer, {
         width: chartContainer.clientWidth,
-        height: 450,
-
-        // TODO: timescale is not visible
+        height: 500,
+        //MODIFY THEME COLOR HERE
+        layout: {
+          background: { color: "#191B1D" }, //base-100
+          textColor: "#fff", //primary-content
+        },
+        //MODIFY THEME COLOR HERE
+        grid: {
+          vertLines: { color: "#292C34" }, //neutral
```

Comment:
@nguvictor we don't use the neutral mostly, maybe base-200? also please review https://github.com/DeXter-on-Radix/website/pull/128 I'm updating colors there to reflect figma more.

Maybe use js import (import the js object from `tailwind.config.js` instead of hard-copying?

Reply:
Base-200 wasn't visible even after adjusting to #191B1D found in #128 
![image](https://github.com/DeXter-on-Radix/website/assets/3978285/c772fe92-faf9-411a-886c-c9a520765ef3)


------
primary-content was way too strong
![image](https://github.com/DeXter-on-Radix/website/assets/3978285/e173996e-8f53-4c31-9514-e7bbe8080592)

------
secondary-content seems about right 
![image](https://github.com/DeXter-on-Radix/website/assets/3978285/7677b6b7-866b-447c-a16d-0537c69228f2)